packages/pass-style/src/types.d.ts

```diff
@@ -78,7 +79,9 @@ export type PassByCopy = Atom | Error | CopyArray | CopyRecord | CopyTagged;
 
 export type PassByRef =
```

Comment:
Conceptually is this the same as `PassableCap`? Do we need both? When should we use one vs the other? Does `PassableCap` need the same elaboration?

Reply:
I am not interested in renaming or cleaning up existing definitions in this PR. It does seem to exclude Promise.

I have some attempts at properly recursing promise type narrowing in PassableCaps, maybe some day I'll be able to land it in a way that doesn't make agoric-sdk explode.

Reply:
`PassByRef` is different and stricter than `PassableCap`. I don't think it should be deprecated. There is some cleanup we could do, possibly making `PassableCap` stricter or in fact make `Passable` a more customizable type. There has been attempts before, one of them at https://github.com/endojs/endo/pull/2421. I suppose we don't have a proper issue that documents the problem.

Reply:
Ok, reluctantly resolving. Such an issue would be appreciated though.